pyproject.toml

```diff
@@ -51,7 +51,7 @@ flake8-bugbear = "^20.1.4"
 
 
 [tool.poetry.extras]
-pydantic = ["pydantic", "typing_extensions"]
+pydantic = ["pydantic", "typing_extensions", "pydantic[email]"]
```

Comment:
What does Pydantic[email] brings? 

Anything else other than EmailStr?

Reply:
just that